.github/workflows/azure_automation_test.yml

```diff
@@ -85,10 +102,25 @@ jobs:
           echo "service-account=${{ inputs.service-account          || 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com' }}" >> "$GITHUB_OUTPUT"
           echo "log-results=true" >> $GITHUB_OUTPUT
           echo "test-context=azure-e2e" >> $GITHUB_OUTPUT
+          # This env param is used to normalize a `workflow_dispatch` param into an env var that always exists
+          ENV=local
+          BEE_NAME_TEMP=${{ env.BEE_NAME }}
+          IS_TRIGGERED_FROM_PR_DISPATCH=false
+          if ${{ github.event_name == 'workflow_dispatch' }}; then
+            ENV=${{ inputs.terra-env }}
+            IS_TRIGGERED_FROM_PR_DISPATCH=${{ inputs.is-triggered-from-pr-dispatch }}
+            if ${{ inputs.is-triggered-from-pr-dispatch == 'true' }}; then
+              BEE_NAME_TEMP=${{ inputs.bee-name }}
+            fi
+          fi
```

Comment:
because this is a shell ... 
do we need to wrap this in any way? we are intermixing GitHub expressions with shell scripting